### ISSUE TITLE
Graduate Teleport Hotspot and add Unit Tests

### ISSUE THREAD

**Issue by RogPodge** [microsoft/MixedRealityToolkit-Unity#9515]
## Describe the problem

When graduating the Teleport System, we brought in the Teleport Hotspot script without Unit tests and Documentation.

## Describe the solution you'd like

Test and refine the Teleport Hotspot and add Unit tests and Documentation around it.

**Comment by polar-kev** [comment]
Documentation is now captured in #9519

**Comment by RogPodge** [comment]
Phase 1 of this located here, waiting on feedback from the design team for making this feature more useful for users, in addition to potentially adding new UI/UX elements for navigation: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9522

**Comment by IssueSyncBot** [comment]
We appreciate your feedback and thank you for reporting this issue.

Microsoft Mixed Reality Toolkit version 2 (MRTK2) is currently in limited support. This means that Microsoft is only fixing high priority issues. Unfortunately, this issue does not meet the necessary priority and will be closed. If you strongly feel that this issue deserves more attention, please open a new issue and explain why it is important.

Microsoft recommends that all new HoloLens 2 Unity applications use MRTK3 instead of MRTK2.

Please note that MRTK3 was released in August 2023. It features an all-new architecture for developing rich mixed reality experiences and has a minimum requirement of Unity 2021.3 LTS. For more information about MRTK3, please visit [https://www.mixedrealitytoolkit.org](https://www.mixedrealitytoolkit.org).

Thank you for your continued support of the Mixed Reality Toolkit!
